scripts/vm/hypervisor/kvm/nsrkvmbackup.sh

```diff
@@ -0,0 +1,270 @@
+#!/bin/bash
+
+## Licensed to the Apache Software Foundation (ASF) under one
+## or more contributor license agreements.  See the NOTICE file
+## distributed with this work for additional information
+## regarding copyright ownership.  The ASF licenses this file
+## to you under the Apache License, Version 2.0 (the
+## "License"); you may not use this file except in compliance
+## with the License.  You may obtain a copy of the License at
+##
+##   http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing,
+## software distributed under the License is distributed on an
+## "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+## KIND, either express or implied.  See the License for the
+## specific language governing permissions and limitations
+## under the License.
+
+
+version=0.9.1
+
+
+OPTIND=1
+
+verb=0
+logDir="/nsr/logs/cloudstack/"
```

Comment:
should this be /var/log/cloudstack/management/... ?

Reply:
Since this is basically talking with EMC Networker client administrators expecting to find logs at the default /nsr/logs directory.